xds/src/main/java/io/grpc/xds/CdsLoadBalancer.java

```diff
@@ -274,11 +272,16 @@ protected Helper delegate() {
 
     @Override
     public void onClusterChanged(ClusterUpdate newUpdate) {
-      channelLogger.log(
-          ChannelLogLevel.DEBUG, "CDS load balancer received a cluster update: {0}",  newUpdate);
+      if (logger.isLoggable(XdsLogLevel.INFO)) {
+        logger.log(
+            XdsLogLevel.INFO,
+            "Received cluster update from xDS client {0}: "
```

Comment:
Override `toString()` for `ClusterUpdate`, then
 ```java
"Received cluster update from xDS client:\n{0}", newUpdate
```

Reply:
For INFO level log, I would prefer to not print out object directly. Another reason is that, I saw Sanjay is adding other fields to `ClusterUpdate`, those are not things we are interested in. It's fine to print them in DEBUG level logs (since we know what's going on), but for INFO level logs, they should not be shown.